src/PostgREST/OpenAPI.hs

```diff
@@ -183,9 +184,14 @@ makeRowFilters tn = map (makeRowFilter tn)
 makePathItem :: (Table, [Column], [Text]) -> (FilePath, PathItem)
 makePathItem (t, cs, _) = ("/" ++ unpack tn, p $ tableInsertable t)
   where
+    -- Use first line of table description as summary; rest as description (if present)
+    -- We strip leading newlines from description so that users can include a blank line between summary and description
+    (tSum, tDesc) = fmap fst &&& fmap (dropWhile (/='\n') . snd) $
+                    breakOn "\n" <$> tableDescription t
     tOp = (mempty :: Operation)
       & tags .~ Set.fromList [tn]
-      & description .~ tableDescription t
+      & summary .~ tSum
```

Comment:
I see, so in case somebody adds a SQL comment to the object, and that comment is the empty string, then we'll preserve it as such. Even though it's a little weird, it's what the person specified in their SQL.